modules/tinymce/src/core/test/ts/browser/PadEmptyWithBrTest.ts

```diff
@@ -0,0 +1,33 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinySelections, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.core.EditorPadEmptyWithBrTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    add_unload_trigger: false,
+    disable_nodechange: true,
+    custom_elements: 'custom1,~custom2',
+    extended_valid_elements: 'custom1,custom2,script[*]',
+    entities: 'raw',
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce',
+    pad_empty_with_br: true,
+    remove_trailing_brs: false,
```

Comment:
Should we have some tests when `remove_trailing_brs` is set to `false` as well for sanity that we are getting the expected behaviour?